mod/wet4/actions/object/question/autocomplete.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
Do we want to put this action in the questions mod if we are going to maintain it.